### DISCUSSION TITLE
Licence locked

### DISCUSSION THREAD

**Discussion by jjh63084** [aclap-dev/video-downloadhelper#1223] [Help and Q&A]
### Your question

I recently changed my internet provider and Download Helper keeps erasing my license key making me have to re-enter it. Now I can't even re-enter my key, it says License locked (revalidating). How do I resolve this issue?

### Addon & CoApp version and details

I recently changed my internet provider and Download Helper keeps erasing my license key making me have to re-enter it. Now I can't even re-enter my key, it says License locked (revalidating). How do I resolve this issue?

**Comment by jcfrog** [comment]
DownloadHelper don't erase your license. It's something on your config, like maybe browser settings.

If your license is locked, you should receive an email to unlock it. If you don't, please send us an email at licenses@downloadhelper.net, we will unlock it for you.

**Comment by jcfrog** [comment]
might be this
<img width="967" alt="Capture d’écran 2024-08-16 à 09 19 30" src="https://github.com/user-attachments/assets/8cf2f220-c966-48b6-bff3-fba67951de99">
